conference.js

```diff
@@ -219,7 +220,16 @@ function maybeRedirectToWelcomePage(options) {
     if (config.enableWelcomePage) {
```

Comment:
The WelcomePage was rewritten in React i.e. it was moved to the folder react/features so it looks to me like the source code bellow needs to be rewritten to take into account that now the React App is abstracting the relation between window.location and the React Component rendered at that window.location using Route (an abstraction similar to react-router's).